common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -768,6 +768,10 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     get(PARTITION_SORTER_DIRECT_MEMORY_RATIO_THRESHOLD)
   def workerDirectMemoryPressureCheckIntervalMs: Long = get(WORKER_DIRECT_MEMORY_CHECK_INTERVAL)
   def workerDirectMemoryReportIntervalSecond: Long = get(WORKER_DIRECT_MEMORY_REPORT_INTERVAL)
+  def workerDirectMemoryTrimChannelWaitInterval: Long =
```

Comment:
This variable is not used and can be removed.

Reply:
It is useful